### PR TITLE
Refactor: Make if-else case statement

### PR DESCRIPTION
**Pull request by TobiPeterG**
As requested in https://github.com/openSUSE/sdbootutil/pull/62

### CODE REVIEW

File: sdbootutil [hunk 1]
```diff
@@ -3109,75 +3109,80 @@ if [ "$SDB_ADD_INITIAL_COMPONENT" = "1" ]; then
...
+	remove-all-kernels)
+		remove_all_kernels "${2:-$root_snapshot}" ;;
+	set-default-snapshot)
+		set_default_snapshot "${2:-$root_snapshot}" ;;
+	is-installed)
+		if is_installed; then
+			log_info "systemd-boot was installed using sdbootutil"
+			exit 0
+		else
+			g_info "not installed using this tool"
```
Comment (aplanas) [inline sdbootutil:3145]: `log_info`

**Review by aplanas — CHANGES_REQUESTED** [review]
small typo but LGTM. Thanks!!

File: sdbootutil (hunk 1 above)
Comment (TobiPeterG) [reply sdbootutil]: whoops missed that, fixed :)

BTW. thank you for the quick review :)
However, it is Saturday. so why are you working ;)

**Review by aplanas — APPROVED** [review]

File: sdbootutil (hunk 1 above)
Comment (aplanas) [reply sdbootutil]: : D

You were working very late yesterday!

**Pull request merged**